sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/FieldsShouldNotBePublic.CSharp9.cs

```diff
@@ -0,0 +1,27 @@
+﻿public record PublicClass
```

Comment:
FieldShouldBeReadonly and FieldsShouldBeReadonly is quite confusing

could we rename FieldsShouldNotBePublic to FieldsShouldBeEncapsulatedInProperties ? 

I left a message on https://jira.sonarsource.com/browse/RSPEC-1104 whether RSPEC-2357  should be deprecated in favor to RSPEC-1104